### PR TITLE
Get entra policy

### PR DESCRIPTION
**Pull request by v-uansari**

### CODE REVIEW

**Review by SteveMutungi254 — DISMISSED** [review]
LGTM.

File: module/Entra/AdditionalFunctions/Get-EntraPolicy.ps1 [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # ------------------------------------------------------------------------------
 #  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 # ------------------------------------------------------------------------------
-
```
Comment (KenitoInc) [inline module/Entra/AdditionalFunctions/Get-EntraPolicy.ps1:4]: We should have kept this blank line

File: test/module/Entra/Get-EntraPolicy.Tests.ps1 [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # ------------------------------------------------------------------------------
 #  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 # ------------------------------------------------------------------------------
-
```
Comment (KenitoInc) [inline test/module/Entra/Get-EntraPolicy.Tests.ps1:4]: We should have kept this line

File: test/module/EntraBeta/Get-EntraBetaPolicy.Tests.ps1 [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # ------------------------------------------------------------------------------
 #  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 # ------------------------------------------------------------------------------
-
```
Comment (KenitoInc) [inline test/module/EntraBeta/Get-EntraBetaPolicy.Tests.ps1:4]: We should have kept this line

File: module/Entra/AdditionalFunctions/Get-EntraPolicy.ps1 (hunk 1 above)
Comment (v-uansari) [reply module/Entra/AdditionalFunctions/Get-EntraPolicy.ps1]: updated

File: test/module/Entra/Get-EntraPolicy.Tests.ps1 (hunk 2 above)
Comment (v-uansari) [reply test/module/Entra/Get-EntraPolicy.Tests.ps1]: updated

**Review by SteveMutungi254 — DISMISSED** [review]
LGTM.

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Pull request merged**